### PR TITLE
Add dependency entry for swapi l3b combined

### PR DESCRIPTION
**Pull request by pleasant-menlo**
# Change Summary
Add an entry to the dependencies csv for swapi l3b combined
Ted and Peter from Menlo

## Overview
Add an entry to the dependencies csv for swapi l3b combined

## New Dependencies
None

## New Files
None

## Deleted Files
None

## Updated Files
- dependency_config.csv
   - add entry for swapi l3b combined

## Testing
None

### CODE REVIEW

**Comment by pleasant-menlo** [comment]
We are not sure how to add a reviewer or assign the PR to ourselves. Please let us know if there are more steps that we need to take.

File: sds_data_manager/lambda_code/SDSCode/dependency_config.csv [hunk 1]
```diff
@@ -111,6 +111,7 @@ swapi, l1, sci, swapi, l2, sci, HARD, DOWNSTREAM
 
 swapi, l2, sci, swapi, l3a, alpha-sw, HARD, DOWNSTREAM
 swapi, l2, sci, swapi, l3a, proton-sw, HARD, DOWNSTREAM
+swapi, l2, sci, swapi, l3b, combined, HARD, DOWNSTREAM
```
Comment (tech3371) [inline sds_data_manager/lambda_code/SDSCode/dependency_config.csv:114]: Is SWAPI l2 data still an input to l3b data product or are l3a products input to l3b data product?

Comment (pleasant-menlo) [reply sds_data_manager/lambda_code/SDSCode/dependency_config.csv:114]: Yes, the l3b data product is calculated directly from the l2 data and does not depend on l3a.
- Ted and Peter

Comment (tech3371) [reply sds_data_manager/lambda_code/SDSCode/dependency_config.csv:114]: cool. I wanted to check this before approving.

**Review by tech3371 — APPROVED** [review]

**Comment by tech3371** [comment]
Can you rebase this branch with latest changes? Then I can merge this PR.

**Comment by pleasant-menlo** [comment]
We have rebased the branch.

**Pull request merged**